packages/acisfp_check/test_regress.sh

```diff
@@ -1,4 +1,6 @@
 acisfp_check \
    --outdir=out \
-   --oflsdir=/data/acis/LoadReviews/2018/MAY2818/oflsa \
-   --run-start=2018:142
+   --oflsdir=${SKA}/data/ska_testr/test_loads/2019/MAY2019/oflsa \
```

Comment:
My thought was that we'd just make a tarball of these test_loads (the one test load) but not sure if we should require a PR to add to ska_sync before promoting this PR.

Reply:
A PR to ska_sync would be fine and can be quick enough.  And not to over-optimize, but I think that the vast majority (by file size) of what's in the standard load products is not needed to run load review.

Reply:
OK.  In the new ska_testr data I made for this one week, I copied over only the minimal set of files starcheck and the acis checkers required anyway.

Reply:
Looks good.